applauncher/004-three-by-three.qml

```diff
@@ -150,13 +98,6 @@ Item {
                 currentPressedIndex = index
             }
 
-            Connections {
-                target: appLauncher
-                function onFakePressedChanged() {
```

Comment:
This fake pressed variable is also an implicit global variable, it is used to implement the tutorial.

I guess we should question whether any launcher apart from the 000 default launcher should have logic for the tutorial... But let's keep separately

Reply:
Restored.